docs/CONFIGURATION.md

```diff
@@ -85,6 +80,16 @@ Specify the thresholds for mutation scores.
 
 Setting `break=0` (default value) ensures that the build will never fail.
 
+## Process runner config
+#### test-runner-command
+
+**Config file:** `test-runner-command: { command: "sbt", args: "test" }`    
+**Mandatory:** Yes  
+**Description:**  
+With `test-runner-command` you specify how stryker4s can invoke the test runner. 
+Examples would be `sbt test`, `mvn test` or any other command to run your tests, including any parameters your tests might need.
```

Comment:
We should mention that the preferred way is through a plugin, both for performance and ease of use reasons. But should the project not be supported, this also works (but will be slower). And also make very clear that this only applies to running with the process-runner, not the plugins.